### PR TITLE
Adds link to the Guild's envelop docs with info for secure GraphQL api

### PR DESCRIPTION
**Pull request by dthyresson**
Envelop has documentation with best practices for securing your GraphQL API.

This PR links to it for further reading.

### CODE REVIEW

**Review by dac09 — CHANGES_REQUESTED** [review]
Link is now 404-ing :)

File: docs/graphql.md [hunk 1]
```diff
@@ -485,6 +485,8 @@ By logging the operation name and extracting the duration for each query, you ca
 
 We'll document more GraphQL security best practices as Redwood reaches a `v1.0` release candidate. For now, know that Redwood already has some baked-in best practices; for example, when deploying GraphQL to production, GraphQL Playground is automatically disabled. 
 
+For more information about how to keep your GraphQL API secure, please read [Securing your GraphQL API](https://www.envelop.dev/docs/guides/) from [the Guild](https://the-guild.dev).
```
Comment (thedavidprice) [inline docs/graphql.md:488]:
```suggestion
For more information about how to keep your GraphQL API secure, please read [Securing your GraphQL API](https://www.envelop.dev/docs/guides/securing-your-graphql-api) from [the Guild](https://the-guild.dev).
```

Looks like you'll need to use this link @dthyresson 

Go ahead an merge once you're happy with it.

**Comment by dac09** [comment]
Hey @dthyresson - so I did a quick skim of the guid docs. They're very solid docs - but they seem to go in a different direction to how we do auth in Redwood. Worth reviewing again before merge!

**Comment by dthyresson** [comment]
Yeah, on second thought this muddies the issue. I am closing without merging.

**Pull request closed without merging**